### PR TITLE
update ML model

### LINKED ISSUE #116: Programmatically tag the dataset based on the exploration from #108 and create classification algorithm for feedback
# User Story:
From the exploration done in #108 we determined that 306 of the feedback entries are "good" feedback. We should tag those as "good" in the data set and tag all remaining as "bad". 

Use this new dataset to create a classification algorithm for any new feedback that is submitted.

306 - good feedback
14694 - bad feedback

You need to split the data 70/30 but randomly. Train the classification on the 70 and test on the 30. Accuracy of the model at the end.

# Acceptance Criteria
- [x] Tag those as "good" in the data set and tag all remaining as "bad".
- [x] Split the data 70/30 randomly, 30% good feedback and 30% bad feedback for training data, and the rest be the testing data.
- [x] Show the accuracy score of the model.

### PR DESCRIPTION
**Pull request by zaks2318**
# Documentation:

Closes #116 

Add some code to the file that tag the dataset with "good" and "bad" and store in temporary list.
Also, split the data 70/30 randomly with 30% "good" feedback and 30% "bad" bad feedback for training data, and the rest for testing data.
Last, show the accuracy of the model.

User story, acceptance criteria will show on the [related issue](https://github.com/North-Seattle-College/ad440-winter2022-tuesday-repo/issues/116)

| Task Breakdown Description      | Approx. Time |
| ----------- | ----------- |
| Researching for LinearRegression | 1 hour       |
| coding for extract and clean data | 2 hours        |
| Frontend Team Meets | 1 hours        |

## How to deploy the functionality locally or in the cloud?
It can run locally after download the file.

## How to configure the functionality?
It needs to use Jupyter notebook, pandas, NumPy, nltk, matplotlib, sklearn, and seaborn python libraries.
Thus, you can run it after downloading them with [pip](https://pip.pypa.io/en/stable/).
Also, we use "floopData02-16-2022-21-06-37.json" as data, so go to AWS S3 dashboard and the file is in the bucket called floop-dataset.

## How to access the implemented functionality?

Open the file in Jupyter notebook, and run each grid.

## What input to provide?

The input will be floopData02-16-2022-21-06-37.json file.

## What output is expected?

The output will be DataFrames  and graphics of the floopData02-16-2022-21-06-37.json.
Also, the prediction and knn.score. 

## Initial estimated time
5 hours

### CODE REVIEW

**Review by peterTorres7 — APPROVED** [review]
I was able to run and test ML model on AWS sagemaker, nicely done!

**Review by ChristopherNavo — APPROVED** [review]
ML model test ran accordingly on SageMaker.  Great Job @zaks2318 (JiaJun)!!
